arbitrator/wasm-libraries/user-host-trait/src/lib.rs

```diff
@@ -534,7 +534,7 @@ pub trait UserHost<DR: DataReader>: GasMeteredMachine {
     fn return_data_size(&mut self) -> Result<u32, Self::Err> {
         self.buy_ink(HOSTIO_INK)?;
         let len = *self.evm_return_data_len();
-        trace!("return_data_size", self, be!(len), &[], len)
+        trace!("return_data_size", self, &[], be!(len), len)
```

Comment:
Why don't I see a signature change for the trace! macro in this PR?
Was it modified in an earlier PR?

Reply:
The `trace` macro was already working correctly on the Stylus side. This PR adds the trace handling on the host/nitro side.

Regarding this change, I swapped the arguments and outputs to better reflect the semantics of the fields. The return value was being sent as an argument instead of an output. It wasn't a problem until now because this was not being used before,

Reply:
I opened a PR on cargo stylus updating the API: https://github.com/OffchainLabs/cargo-stylus/pull/73